<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BillDetailItemEditSubPage</class>
 <widget class="QWidget" name="BillDetailItemEditSubPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>883</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="backgroundwidget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" rowstretch="6,8,12,2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="titlewidget" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(249, 219, 97);
color:black;</string>
        </property>
        <layout class="QGridLayout" name="gridLayout" rowstretch="5,2">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,3,2,1">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="returnButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">border:none;
border-image: url(:/public/image/returnarrowyellowbackground.jpg);</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>108</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_TypeIcon">
             <property name="styleSheet">
              <string notr="true">border-image: url(:/BillPage/image/Expand_1_On_SubPage.jpg);</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>98</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="shareButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">border:none;
font: 17pt &quot;微软雅黑&quot;;</string>
               </property>
               <property name="text">
                <string>分享</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_Type">
           <property name="styleSheet">
            <string notr="true">font: 20pt &quot;微软雅黑&quot;;</string>
           </property>
           <property name="text">
            <string>餐饮</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="midwidget" native="true">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);

QWidget
{
	border:none;
}</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,8">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="styleSheet">
              <string notr="true">color: rgb(185, 185, 185);
font: 18pt &quot;微软雅黑&quot;;
border-bottom:1px ;
border-color: rgba(140,140,140,120);
border-style: solid;</string>
             </property>
             <property name="text">
              <string>类型</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_InAndOut_Type">
             <property name="styleSheet">
              <string notr="true">font: 18pt &quot;微软雅黑&quot;;
border-bottom:1px ;
border-color: rgba(140,140,140,120);
border-style: solid;</string>
             </property>
             <property name="text">
              <string>支出</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,8">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="styleSheet">
              <string notr="true">color: rgb(185, 185, 185);
font: 18pt &quot;微软雅黑&quot;;
border-bottom:1px ;
border-color: rgba(140,140,140,120);
border-style: solid;</string>
             </property>
             <property name="text">
              <string>金额</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_Money">
             <property name="styleSheet">
              <string notr="true">font: 18pt &quot;微软雅黑&quot;;
border-bottom:1px ;
border-color: rgba(140,140,140,120);
border-style: solid;</string>
             </property>
             <property name="text">
              <string>4000</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,8">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="styleSheet">
              <string notr="true">color: rgb(185, 185, 185);
font: 18pt &quot;微软雅黑&quot;;
border-bottom:1px ;
border-color: rgba(140,140,140,120);
border-style: solid;</string>
             </property>
             <property name="text">
              <string>日期</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_Date">
             <property name="styleSheet">
              <string notr="true">font: 18pt &quot;微软雅黑&quot;;
border-bottom:1px ;
border-color: rgba(140,140,140,120);
border-style: solid;</string>
             </property>
             <property name="text">
              <string>2023年1月5日 星期四</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,8">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="styleSheet">
              <string notr="true">color: rgb(185, 185, 185);
font: 18pt &quot;微软雅黑&quot;;
border-bottom:1px ;
border-color: rgba(140,140,140,120);
border-style: solid;</string>
             </property>
             <property name="text">
              <string>备注</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_Remark">
             <property name="styleSheet">
              <string notr="true">font: 18pt &quot;微软雅黑&quot;;
border-bottom:1px ;
border-color: rgba(140,140,140,120);
border-style: solid;</string>
             </property>
             <property name="text">
              <string>烟酒</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QWidget" name="midspacewidget" native="true">
        <property name="styleSheet">
         <string notr="true">border-bottom:1px ;
border-color: rgba(140,140,140,120);
border-style: solid;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QWidget" name="buttombuttonwidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QPushButton" name="editButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">border:none;
font: 18pt &quot;微软雅黑&quot;;
color:black;</string>
           </property>
           <property name="text">
            <string>编辑</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>10</height>
            </size>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <property name="midLineWidth">
            <number>3</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">border:none;
font: 18pt &quot;微软雅黑&quot;;
color:black;</string>
           </property>
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
